tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Certificates/CustomIdentityServerBuilderExtensionTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using FluentAssertions;
+using IdentityServer4.Stores;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using NHSD.BuyingCatalogue.Identity.Api.Certificates;
+using NHSD.BuyingCatalogue.Identity.Api.Infrastructure;
+using NHSD.BuyingCatalogue.Identity.Api.Settings;
+using NUnit.Framework;
+using Serilog;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.UnitTests.Certificates
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    internal sealed class CustomIdentityServerBuilderExtensionTests
+    {
+        [Test]
+        public void UseDeveloperCredentials_Sets_DeveloperCredentials()
+        {
+            var settings = new CertificateSettings
+            {
+                UseDeveloperCredentials = true, CertificatePath = "", CertificatePassword = ""
```

Comment:
`string.Empty`?

Reply:
doesn't look like that's been used elsewhere. One for the 2nd bun fight meeting.